src/components/screens/ScheduleScreen.jsx

```diff
@@ -35,52 +36,62 @@ const ScheduleScreen = () => {
   const suppliersToBeCreated = useSelector(getSuppliersNotInContentful);
 
   useEffect(() => {
-    if (status === PROCESSING_SUPPLIERS) {
+    const contentfulActions = [];
+
+    if (status === PROCESS_SUPPLIERS) {
       suppliersToBeUpdated.forEach((pair) => {
-        updateSupplier(pair, cma)
-          .then(() => {
-            dispatch(
-              setSupplier({
-                supplierId: pair.supplier.id,
-                status: TO_BE_PUBLISHED,
-              }),
-            );
-          })
-          .catch((error) => {
-            dispatch(
-              setSupplier({
-                supplierId: pair.supplier.id,
-                status: CONTENTFUL_PUT_ERROR,
-              }),
-            );
-            console.error(error.message);
-          });
+        contentfulActions.push(
+          updateSupplier(pair, cma)
+            .then(() => {
+              dispatch(
+                setSupplier({
+                  supplierId: pair.supplier.id,
+                  status: TO_BE_PUBLISHED,
+                }),
+              );
+            })
+            .catch((error) => {
+              dispatch(
+                setSupplier({
+                  supplierId: pair.supplier.id,
+                  status: CONTENTFUL_PUT_ERROR,
+                }),
+              );
+              console.error(error.message);
+            }),
+        );
       });
 
       suppliersToBeCreated.forEach((pair) => {
-        createSupplier(pair, cma)
-          .then((result) => {
-            console.log(result);
-            dispatch(
-              setSupplier({
-                supplierId: pair.supplier.id,
-                newContentfulId: result.sys.id,
-                status: TO_BE_PUBLISHED,
-              }),
-            );
-          })
-          .catch((error) => {
-            dispatch(
-              setSupplier({
-                supplierId: pair.supplier.id,
-                status: CONTENTFUL_PUT_ERROR,
-              }),
-            );
-            console.error(error.message);
-          });
+        contentfulActions.push(
+          createSupplier(pair, cma)
+            .then((result) => {
+              console.log(result);
```

Comment:
Can the `console.log` be removed; or is it still needed for something?

Reply:
Oooops! 🙈